### ISSUE TITLE
R 3.6.3 || install crul fails || Mac Mojave

### ISSUE THREAD

**Issue by sariya** [ropensci/crul#129]
Hi there,

I'm on OSX (Mojave 10.14.1), R 3.6.3, using install.packages. It fails with following error:

```
Warning: unable to access index for repository https://nbcgib.uesc.br/mirrors/cran/src/contrib:
  cannot open URL 'https://nbcgib.uesc.br/mirrors/cran/src/contrib/PACKAGES'
Warning message:
package ‘crul’ is not available (for R version 3.6.3) 
```

**Comment by sckott** [comment]
i don't think this is an issue with `crul` itself. can you install any package from cran?

**Comment by sariya** [comment]
Oh, Ah! not able to install `stringi` either, for example. 
Fails with following error:

```
Warning: unable to access index for repository https://nbcgib.uesc.br/mirrors/cran/src/contrib:
  cannot open URL 'https://nbcgib.uesc.br/mirrors/cran/src/contrib/PACKAGES'
Warning message:
package ‘stringi’ is not available (for R version 3.6.3) 
```
